actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Use `path_parameters_utf8_enforced` instead of `path_parameters`
```

Comment:
I think this change log info focus too much on the implementation details. How about:
'Force path parameters to be UTF8'